modyn/tests/evaluator/internal/grpc/test_evaluator_grpc_servicer.py

```diff
@@ -161,9 +155,8 @@ def get_evaluation_info(evaluation_id, model_path: pathlib.Path, config: dict):
 
 
 @patch.object(EvaluatorGRPCServicer, "connect_to_storage", return_value=DummyStorageStub())
-@patch.object(EvaluatorGRPCServicer, "connect_to_selector", return_value=DummySelectorStub())
 @patch.object(EvaluatorGRPCServicer, "connect_to_model_storage", return_value=DummyModelStorageStub())
```

Comment:
Corresponding to removing the connection to selector, now in tests we also don't need to patch the connection.